### PR TITLE
feat(setting-stylePanel): 样式面板编辑全局样式新UI与交互

### PR DESCRIPTION
**Pull request by chilingling**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/design-core/assets/error.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 14" xml:space="preserve" isSvg="true">
+  <path d="M14 7c0 3.87-3.13 7-7 7s-7-3.13-7-7 3.13-7 7-7 7 3.13 7 7z" style="fill-rule: evenodd; clip-rule: evenodd;"></path>
+  <path d="M9.96 9.29c.09.09.14.21.14.34 0 .13-.05.25-.14.33a.47.47 0 0 1-.67 0L7 7.67 4.71 9.96c-.18.18-.49.18-.67 0a.483.483 0 0 1 0-.67L6.33 7 4.04 4.71c-.09-.09-.14-.21-.14-.34s.05-.25.14-.34c.18-.18.49-.18.67 0L7 6.33l2.29-2.29c.19-.18.49-.18.67 0 .09.09.14.21.14.34s-.05.25-.14.34L7.67 7l2.29 2.29z" style="fill: rgb(255, 255, 255);"></path>
```
Comment (ghost) [inline packages/design-core/assets/error.svg:3]: fill是否改为currentColor，便于样式调整颜色

File: packages/settings/styles/src/Main.vue [hunk 2]
```diff
@@ -125,43 +118,34 @@ export default {
...
-        state.schemaUpdateKey++
-        addHistory()
+      if (styleString) {
+        currentSchema.props.style = styleString
+      } else {
+        delete currentSchema.props.style
       }
+
+      addHistory()
+
```
Comment (ghost) [inline packages/settings/styles/src/Main.vue]: 不必要的空行

File: packages/design-core/assets/error.svg (hunk 1 above)
Comment (chilingling) [reply packages/design-core/assets/error.svg:3]: 这个是 svg 的中间的 "x"，不需要跟随样式颜色调整
image

Comment (ghost) [reply packages/design-core/assets/error.svg:3]: 了解了

File: packages/settings/styles/src/Main.vue (hunk 2 above)
Comment (chilingling) [reply packages/settings/styles/src/Main.vue]: Done

**Review by ghost — APPROVED** [review]
LGTM

File: packages/settings/styles/src/components/classNamesContainer/index.vue [hunk 3]
```diff
@@ -1,113 +1,113 @@
...
+        <div v-if="classNameState.showDropdownList" class="selector-drop-down-list lowcode-scrollbar-thin">
+          <span class="selector-dropdown-list-tips">输入并回车创建新选择器</span>
+          <span v-if="currentSelectorList.length" class="selector-dropdown-list-tips">选择已有选择器编辑</span>
+          <ul class="exist-class-list">
+            <li
+              v-for="item in currentSelectorList"
+              :key="item"
+              :title="item"
+              class="exist-class-item"
+              @mousedown="handleSelectExistingClass(item)"
```
Comment (gene9831) [inline packages/settings/styles/src/components/classNamesContainer/index.vue:79]: 将这里的 `mousedown` 更改为 `click` 是不是更符合直觉点，还有第93行的 `mousedown`

Comment (chilingling) [reply packages/settings/styles/src/components/classNamesContainer/index.vue:79]: 这里无法使用 click 事件，因为 58 行的 blur 监听，会导致 blur 事件比 click 事件先触发，blur 事件触发之后，会收起下拉框，导致无法触发 click 事件，所以需要使用 mousedown 事件进行监听（需要比 blur 事件先执行相关的逻辑）

**Pull request merged**